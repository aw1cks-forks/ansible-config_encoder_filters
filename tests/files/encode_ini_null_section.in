section1:
  var1: some text
  var2: 123
  var3: 987.654
  var4: '!!!null'

section2:
  var5: yes
  var6: "false"
