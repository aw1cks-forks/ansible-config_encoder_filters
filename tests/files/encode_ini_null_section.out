[section1]
var1=some text
var2=123
var3=987.654
var4

[section2]
var5=True
var6=false
